lib/bloc/account/add_funds_bloc.dart

```diff
@@ -11,19 +14,65 @@ class AddFundsBloc {
   final _addFundResponseController = new StreamController<AddFundResponse>();
   Stream<AddFundResponse> get addFundResponseStream => _addFundResponseController.stream;
 
+  final _moonPayOrderController = new StreamController<MoonpayOrder>();
+  Sink<MoonpayOrder> get moonPayOrderSink => _moonPayOrderController.sink;
```

Comment:
Why do we need additional MoonpayOrder controller (sink and stream)?
Can't we use the `_orderController ` also for pending orders?

Reply:
It's removed in [9648661](https://github.com/breez/breezmobile/pull/162/commits/96486619e86debc7d6514689a54adbb2e76a63bd)